doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_arcgis_featuretable_sink.md

```diff
@@ -40,29 +42,75 @@ Argis feature tables must be provisioned before sending entities.
 #### <a name="section1.2.1"></a>ArcGis databases naming conventions
 Each entity type needs an url and an unique field to be persisted into the feature table.
 
-NGSIArcgisFeatureTableSink composes each table's url with entitie's `service` and `service path`, to provide multiple tables access. 
+NGSIArcgisFeatureTableSink composes each table's url with entitie's `service` and `service path`, to provide multiple tables access. The final url is compounds by `cygnus-ngsi.sinks.arcgis-sink.arcgis_service_url`+`fiware-service`+`fiware-servicepath`. 
 
-Unique field is provided to allow `NGSIArcgisFeatureTableSink` to update existant entities. NGSI `entity type` will be used as unique field name.
+Unique field is provided to allow `NGSIArcgisFeatureTableSink` to update existant entities. NGSI `entity type` will be used as unique field name. This means that a feature named `type` in the Feature Table cannot be filled in by the sink. If Feature Table needs to persist the value of entity type it has to be in a feature different than `type`.
 
-All this parameters, can be customized using Cygnus mapping capabilities.
+All this parameters, can be customized using Cygnus mapping capabilities or Context Broker custom notifications.
+
+##### Using Cygnus
 
 Let's see an example:	
 
-##### Agent.conf file:
+###### Agent.conf file:
 
 	agent.arcgis-sink.arcgis_service_url = https://arcgis.com/{hash}/arcgis/rest/services
-##### Entity data:
+###### Entity data:
 
 	service = vehicles
 	service-path = /4wheels
 	entity-type = car
-##### result
+###### result
 
 	Feature table url: https://arcgis.com/{hash}/arcgis/rest/services/vehicles/4wheels
 	Table's unique field: car
 
 [Top](#top)
 
+#### Using Context Broker custom notifications
```

Comment:
Should be included in ToC above? It seem that four level sections (i.e. `####`) are included in ToC, if I have understood correctly.

Reply:
The title level was wrong, fixed in https://github.com/telefonicaid/fiware-cygnus/pull/2327/commits/deb80da960593b8e59aed2dbddfb0cc167f0edf4